doc/architecture/context.md

```diff
@@ -58,20 +63,32 @@ Global value and local values in the Soil layer:
 
 ## Plant Database
 
-- photos
-- genus and species
-- other general infos
-- size (height, width, depth)
+- hierarchy: family, subfamily, genus, species, variety
+- pointer to one hierarchy up (MISSING)
+- if it is abstract/concrete (MISSING: we assume all plants of Practical Plants to be concrete)
+- common English name
+- common German name
+- icon (MISSING)
+- dates (MISSING):
+  - begin/end raising
+  - begin/end planting
+  - begin/end seeding
+  - begin/end harvest
+- size (height, width)
 - constraints where they want to be:
-  - recommended distances to other plants
+  - good/bad neighbors to other specific plants of any hierarchy (MISSING)
+  - recommended distances to any other plant, needed for area calculations (MISSING)
   - sun
   - soil weight
   - pH value of topsoil
-  - yield grade
-  - freeze resistance
-  - good/bad neighbors
+  - nutrition demand (MISSING, start with "Nutritionally poor soil" in "Environmental Tolerances")
```

Comment:
@markus2330 could you please specify the rule here? As a reminder, we have an enum with 'light feeder', 'moderate feeder', and 'heavy feeder'.

Reply:
If "Nutritionally poor soil" is given 'light feeder' should be set.